nilearn/plotting/displays/_projectors.py

```diff
@@ -62,6 +62,71 @@ def draw_cross(self, cut_coords=None, **kwargs):
         """
         pass
 
+    def input_checks(
```

Comment:
```suggestion
    def _check_inputs_add_graph(
```

- leading underscore to denote this is a private method
- more specific name (we may end up having several input_checks method in the end)

Reply:
Fair enough. Noted.